contracts/truefi2/TrueLender2.sol

```diff
@@ -399,23 +296,6 @@ contract TrueLender2 is ITrueLender2, UpgradeableClaimable {
         return borrowSum;
     }
 
-    /**
```

Comment:
The `totalBorrowed()` function in line 381 was only ever called by our new credit model, so we can recursively remove that too. If you prefer, we can do this in a separate PR, although I think it fits here and should be pretty straightforward.